server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -184,7 +184,7 @@ export class MetadataExtractionProcessor {
       asset = await this.assetCore.save({ id: asset.id, fileCreatedAt: fileCreatedAt?.toISOString() });
       await this.jobRepository.queue({ name: JobName.STORAGE_TEMPLATE_MIGRATION_SINGLE, data: { asset } });
     } catch (error: any) {
-      this.logger.error(`Error extracting EXIF ${error}`, error?.stack);
+      this.logger.error(`Error extracting EXIF ${error} on file ${asset.originalPath}`, error?.stack);
```

Comment:
I think in all of these jobs (that do operations on a file) we should log (1) asset id, (2) filename/path, (3) error + stack.